src/resolvers.ts

```diff
@@ -120,11 +143,29 @@ export async function resolveLegacyDclUrl(uri: URL) {
     path = uri.pathname.replace(/^\//, '').split('/')
   }
 
+  const tokenIdPattern = /^[0-9]+$/
```

Comment:
Hey, I think these token ids can be alpha-numeric as well. Commenting here so we don't forget about this 👼